dynamo/preprocessing/external/sctransform.py

```diff
@@ -443,5 +443,5 @@ def sctransform(
                 X_squared = adata.X.copy()
                 X_squared.data **= 2
                 variance = X_squared.mean(0) - np.square(adata.X.mean(0))
-                adata.var["sct_score"] = variance.toarray().ravel() if sp.issparse(variance) else variance.ravel()
+                adata.var["sct_score"] = variance.A1
```

Comment:
can you please double check if this line would work properly if variance is a sparse matrix

Reply:
since you've validated variance is not sparse, i've no additional comments